migrate/20231107_add_basic_fw.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:firewall_rule) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
```

Comment:
I think we should use .create_with_id instead.

Reply:
That's right, fixed it.